pkgs/top-level/all-packages.nix

```diff
@@ -10198,6 +10198,7 @@ with pkgs;
 
   v8 = callPackage ../development/libraries/v8 {
     inherit (python2Packages) python gyp;
+    libtool = darwin.cctools;
```

Comment:
Sure, I followed what I found already in the file, but that would make more sense.